exec/java-exec/src/main/java/org/apache/drill/exec/ops/ContextInformation.java

```diff
@@ -28,12 +28,14 @@
   private final String currentDefaultSchema;
   private final long queryStartTime;
   private final int rootFragmentTimeZone;
+  private final int hllMemoryLimit;
 
   public ContextInformation(final UserCredentials userCredentials, final QueryContextInformation queryContextInfo) {
     this.queryUser = userCredentials.getUserName();
     this.currentDefaultSchema = queryContextInfo.getDefaultSchemaName();
     this.queryStartTime = queryContextInfo.getQueryStartTime();
     this.rootFragmentTimeZone = queryContextInfo.getTimeZone();
+    this.hllMemoryLimit = queryContextInfo.getHllMemoryLimit();
```

Comment:
Re-labeled as hll_accuracy to avoid confusion. This option is meant to control the accuracy of NDV computation. However, more accuracy comes at the cost of using more memory.